test/ros3.c

```diff
@@ -865,7 +865,7 @@ test_vfd_open(void)
         if (T.which_fapl == FAPL_FILE_ACCESS)
             _fapl_id = fapl_file_access;
         else if (T.which_fapl == FAPL_ROS3_ANON)
-            _fapl_id = fapl_id;
+            _(void) fapl_id;
```

Comment:
?

Reply:
whoops. good catch. A find-replace screwup I guess. Clearly I'm not compiling that file. Do you know what build option I'm missing?

Reply:
> whoops. good catch. A find-replace screwup I guess. Clearly I'm not compiling that file. Do you know what build option I'm missing?

I believe that the "read-only s3" ("ros3") VFD needs to talk to an S3 server, but it's enabled with the --enable-ros3-vfd configure flag.

Reply:
Fixed.